modules/lib/src/main/resources/assets/js/app/ai/ui/InteractionType.ts

```diff
@@ -0,0 +1 @@
+export type InteractionType = 'click';
```

Comment:
Not sure that it deserves to be in a separate file (even more so under `/ui`). We won't probably use it as import, so it can be just an inner type in AiContentOperatorInteractionEvent